src/Tiles.py

```diff
@@ -126,38 +129,54 @@ def asset(self):
 
 
 class GameTile:
-    def __init__(self, number_label, tile, points, tile_id):
+    def __init__(self, number_label, real_number, tile, points, tile_id):
```

Comment:
Correct me if i'm wrong but i don't think we need this change because that's exactly what the argument `number_label` is for right?

Reply:
Can quite remember but is has something to do with the number_label not matching the numbers seen on the board

Reply:
🤔 okay not sure then I'll have another look at it later